libraries/FlashStorage/src/FlashStorage.h

```diff
@@ -35,10 +35,14 @@ template <uint32_t _storage_size,
           uint32_t _fmc_end = ARDUINO_UPLOAD_MAXIMUM_SIZE>
 class FlashStorage
 {
+    static_assert(_storage_size % 4096 == 0,
```

Comment:
In theory, should this ideally be 4096 if we're in BANK1 and 2048 if BANK0? (If it is, we probably want a comment)

Reply:
Yes, ideally it should be 2048 if we're in bank0, but calculating where we are, at compile time, so we can assert correctly is more trouble than worth. I'll add a comment explaining that, thanks!